### PR TITLE
[ISSUE#7291] Code Cleanliness needs improvement in NamingHttpClientPr…

### PR DESCRIPTION
**Pull request by CaiJiHuzhuo**
…oxy.java (#7291)

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Code Cleanliness needs improvement in NamingHttpClientProxy

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by li-xiao-shuang — APPROVED** [review]

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java [hunk 1]
```diff
@@ -320,6 +320,7 @@ public boolean serverHealthy() {
             String serverStatus = json.get("status").asText();
             return "UP".equals(serverStatus);
         } catch (Exception e) {
+            NAMING_LOGGER.warn("[serverHealthy] Failed to get server health status");
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: Why remove this log？

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java [hunk 2]
```diff
@@ -335,15 +336,9 @@ public ListView<String> getServiceList(int pageNo, int pageSize, String groupNam
...
-            switch (SelectorType.valueOf(selector.getType())) {
-                case none:
-                    break;
-                case label:
-                    ExpressionSelector expressionSelector = (ExpressionSelector) selector;
-                    params.put(SELECTOR_PARAM, JacksonUtils.toJson(expressionSelector));
-                    break;
-                default:
-                    break;
+            if (SelectorType.valueOf(selector.getType()) == SelectorType.label) {
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:339]: should we use switch still but only combine the `none` and `default`?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java (hunk 1 above)
Comment (CaiJiHuzhuo) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: this is add log，not remove。

Comment (KomachiSion) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: sorry, my mistake, but should we print the exception info?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java (hunk 2 above)
Comment (CaiJiHuzhuo) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:339]: Only one condition matches，so choose to use if

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java (hunk 1 above)
Comment (CaiJiHuzhuo) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: i think when request failed，need to print some info，not just return false.

Comment (KomachiSion) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: I know, I means print more information for exception detail

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java (hunk 1 above)
Comment (CaiJiHuzhuo) [reply client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:323]: oh , yes，need to print exception

**Comment by KomachiSion** [comment]
any process? If no more changes and response, we will close this PR>

**Comment by KomachiSion** [comment]
No more response from author.

**Pull request closed without merging**